### PR TITLE
docs: add sample for Cloud Run Secure Services tutorial

### PR DESCRIPTION
**Pull request by grayside**
This PR creates a new Terraform example showing how to create the app stepped through in https://cloud.google.com/run/docs/tutorials/secure-services.

This PR loosely followed patterns from #6204 and guidance from @pattishin. 

No release notes are needed.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by grayside** [comment]
These errors appear to be the result of cloud run resource names being given a suffix, but that suffix not being injected into references elsewhere in the terraform definition. Is the preferred practice that I add the variables more generally around the template?

File: mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb:2]:
```suggestion
resource "google_cloud_run_service" "renderer" {
```

renderer shouldn't need to be a variable. This is simply a unique identifier for the resource for terraforms.

File: mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb:48]: same

File: mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb:55]: same

File: mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb:23]: same

File: mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb
Comment (grayside) [reply mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb]: Thanks, I've taken this and other comments to mean I shouldn't use the template variable in Terraform resource names, only in Cloud resource names. I've pushed a change with that adjustment.

**Comment by grayside** [comment]
Not sure what to do with these VCR-test errors. Waiting on maintainer feedback.

**Comment by ScottSuarez** [comment]
note the change I commented should fix the failures. You can commit it by clicking `commit suggestion`

**Comment by grayside** [comment]
I applied the change suggested out-of-band, but the tests are failing and I don't see how to parse the test output to find the problem. Waiting on maintainer feedback.

**Comment by ScottSuarez** [comment]
> I applied the change suggested out-of-band, but the tests are failing and I don't see how to parse the test output to find the problem. Waiting on maintainer feedback.

The test you added it now passing. Other failures are unrelated

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**
